shared/team-building/container.tsx

```diff
@@ -199,9 +201,10 @@ const mapDispatchToProps = (dispatch: Container.TypedDispatch, {namespace, teamn
     dispatch(SettingsGen.createRequestContactPermissions({thenToggleImportOn: true})),
   _search: (query: string, service: ServiceIdWithContact, limit?: number) => {
     const func = service === 'keybase' ? debouncedSearchKeybase : debouncedSearch
-    return func(dispatch, namespace, query, service, limit)
+    return func(dispatch, namespace, query, service, namespace === 'chat2', limit)
```

Comment:
this should also be a prop and not some built in thing

Reply:
As in, an OwnProp for the team builder? Why is this different than the code below that looks at `ownProps.namespace === 'teams'`?

Reply:
ok lets leave this as is and just ticket cleaning this up as it shouldn't be below either